### DISCUSSION TITLE
Paper auc unreachable

### DISCUSSION THREAD

**Discussion by masadon129** [gokulkarthik/hateclipper#7] [Q&A]
excuse me.  I'm a college student doing research based on HateCLIPper.  I ran engine.py as run.sh (fusion cross), but I couldn't reach the AUC  in the paper (dev90.98), and the AUC I tried was dev64.59.  I would appreciate if you could tell me this problem and how to solve it.

**Comment by victoriapoghosian** [comment]
hi @masadon129 , were you able to get close results?
